website_sale_cart_no_redirect/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
Should we credit Odoo S.A. being a backport?

Reply:
Yes, it should be co-author of the module, and the README should contain a disclaimer. The license should be also LGPL-3.

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#backporting-odoo-modules

Reply:
Anyways, don't forget that website modules should be LGPL-3 by default for not disturbing custom themes.